junebug/stores.py

```diff
@@ -19,32 +20,44 @@ def __init__(self, redis, ttl=None):
 
     @inlineCallbacks
     def _redis_op(self, func, id, *args, **kwargs):
+        ttl = kwargs.pop('ttl')
+        if ttl is None:
+            ttl = self.ttl
```

Comment:
If `self.ttl = 5`, setting `ttl = None` in the kwargs won't override the `5`? Just checking whether that is intentional?